types/selenium-webdriver/index.d.ts

```diff
@@ -1460,6 +1461,21 @@ export class Capabilities {
  * @interface
  */
 export class Executor {
```

Comment:
Executor is not exported from the index.js file and does not belong here. https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/index.js

It is exported in http.d.ts. It could also be added to a file command.d.ts but it still needs to be exported from http.d.ts. See https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/http.js#L619 and https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/command.js#L198

I think this could be moved to a command.d.ts file then http.d.ts would also have to export the Executor class.

Reply:
Sounds reasonable!
I also added a test for the command executor.